docs/content/direct/examples.md

```diff
@@ -71,24 +72,18 @@ manager which connects to the `wds1` front-end and the `imbs1` OCM control plane
    kflex create wds1 -p kubestellar
    ```
 
-1. Run the OCM based transport controller in a pod.  
-**NOTE**: This is work in progress, in the future the controller will be deployed through a Helm chart.
+1. Run the OCM based transport controller in a pod.
 
```

Comment:
Everything we deploy now runs in a pod, I would just say "Deploy the OCM based transport controller"

Reply:
ok, done